backend/v1/dependencies/database/database_connector.py

```diff
@@ -131,4 +135,11 @@ async def get_species_photo(self, species_name: str):
         :param species_name:
         :return:
         """
-        return self.get_photo(self.FISH_SPECIES_IDENTIFIER_FIELD_NAME, species_name.lower())
\ No newline at end of file
+        return self.get_photo(self.FISH_SPECIES_IDENTIFIER_FIELD_NAME, species_name.lower())
+
+    # async def upload_aquarium_photo(self, aquarium_id: int, file: bytes):
```

Comment:
What about aquarium photos? Will they be implemented in the future?

Reply:
yes today i will be working on it